modules/core/src/core/model.js

```diff
@@ -11,21 +11,13 @@ const ERR_MODEL_PARAMS = 'Model needs drawMode and vertexCount';
 const LOG_DRAW_PRIORITY = 2;
 
 export default class Model extends BaseModel {
-  constructor(gl, props = {}) {
```

Comment:
Hmm... while the change looks sound here, as discussed, this `constructor/initialize/setProps()` pattern is used across luma.gl, we are getting slightly different variants of it in different classes, this is going to be confusing for a while.